src/Utilities/Workspaces/xlf/WorkspacesUtilitiesResources.cs.xlf

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="cs" original="../AnalyzerUtilitiesResources.resx">
+    <body>
+      <trans-unit id="VersionCheckDescription">
```

Comment:
These resources should no longer be used, version check analyzer has been moved to a separate project. I seemed to have forgotten to delete them from utilities.

Reply:
I couldn't find the corresponding RESX file. Should I delete these XLF files altogether, or just prune them somehow?

Reply:
Ah, so seems like I deleted the resx but not the xlf files. I think probably best to just delete the XLF files altogether.

Reply:
I deleted the files as a fourth commit, whenever it gets there 😄

Reply:
Actually I amended the first commit since the build for it hadn't even started.